src/components/CompanyCard/CompanyCard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './companyCard.scss';
+
+
```

Comment:
I've seen you were considering the best approach to pass styles to a component. Probably you meant this one. What would you like to achieve? What kind of styles do you want to pass to the Function element?

Reply:
Did you mean to pass for example a whole file from a path? Or do you want to pass some structure that will be dispatched properly to children nodes inside of the component?

Reply:
like for example:
```

styles = { '.title': { color: 'red', 'font-size': '24px' } }
```

Reply:
The cards are mostly similar. But they also have some subtle differences such as background colors or the flex-direction is reversed. The architecture is the same though. Thats why I was trying to see how to pass some style attributes. Ive found a document. Ill implement and you'll let me know if that works.